.github/workflows/scheduled-triage.yml

```diff
@@ -0,0 +1,38 @@
+name: Triage issues on a schedule
+
+on:
+  push:
+    paths:
+    - .github/workflows/scheduled-triage.yml
+  schedule:
+    # Once every day at midnight UTC
+    - cron: "0 0 * * *"
+
+jobs:
+  scheduled-triage:
+    if: startsWith(github.repository, 'Homebrew/')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Mark/Close Stale Issues and Pull Requests
+        uses: actions/stale@v3
```

Comment:
This should also be triggered on comments, otherwise the `stale` label will just stay there for a whole day.

Reply:
Good idea, thanks

Reply:
Do we really want this to run on all comments? Is there a way to say "run when there's a new comment on an issue with the `stale` label"? Running on every comment feels a little excessive. If a faster turnaround time than one day is desired, we could have it run every n hours

Reply:
Yeah, I guess you can have a condition on the label for the comment trigger.

Reply:
I guess something like 

```yaml
if: >
  github.event_name != 'issue_comment' || (
    contains(github.event.issue.labels.*.name, 'stale') || 
    contains(github.event.pull_request.labels.*.name, 'stale')
  )
```